scripts/background/TabAudioHandler.js

```diff
@@ -25,6 +27,7 @@ function TabAudioHandler() {
             tabUpdatedHandler = checkTabs;
             chrome.tabs.onUpdated.addListener(checkTabs);
             chrome.tabs.onRemoved.addListener(checkTabs); // A tab that is audible can be closed and will not trigger the updated event.
+            checkTabsInterval = setInterval(checkTabs, 100);
```

Comment:
This is used as a fallback for some browsers, including some scenarios in Chrome.